packages/react-toolbar/src/components/ToolbarButton/useToolbarButtonStyles.ts

```diff
@@ -1,7 +1,15 @@
+import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { ToolbarButtonState } from './ToolbarButton.types';
+import type { ToolbarButtonSlots, ToolbarButtonState } from './ToolbarButton.types';
 
+/**
+ * @deprecated
+ */
 export const toolbarButtonClassName = 'fui-ToolbarButton';
```

Comment:
For `ToolbarButton` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```